pkg/api/http_server.go

```diff
@@ -227,6 +227,9 @@ func (hs *HTTPServer) addMiddlewaresAndStaticRoutes() {
 
 	m.Use(middleware.AddDefaultResponseHeaders())
 
+	if setting.AppSubUrl != "" {
+		m.SetURLPrefix(setting.AppSubUrl)
```

Comment:
nice, did not know maracon had this, thought we would have to modify all macaron routes :)